src/chart/onLayout/addControlTooltips.js

```diff
@@ -0,0 +1,43 @@
+export default function addControlTooltips() {
+    const tooltips = {
+        //other controls
+        'Status Group':
```

Comment:
I might like these hovers better than the info bubbles in the heatmap - don't take up space but are still very noticeable

Reply:
Some nice writing here - powerful, concise, clear - moving